minecraft/protocol/packet/add_volume_entity.go

```diff
@@ -5,12 +5,18 @@ import (
 	"github.com/sandertv/gophertunnel/minecraft/protocol"
 )
 
-// AddVolumeEntity sends a volume entity's definition and components from server to client.
+// AddVolumeEntity sends a volume entity's definition and metadata from server to client.
 type AddVolumeEntity struct {
-	// EntityRuntimeID ...
+	// EntityRuntimeID is the runtime ID of the entity. The runtime ID is unique for each world session, and
+	// entities are generally identified in packets using this runtime ID.
 	EntityRuntimeID uint64
-	// VolumeEntityData ...
-	VolumeEntityData map[string]interface{}
+	// EntityMetadata is a map of entity metadata, which includes flags and data properties that alter in
+	// particular the way the entity looks.
+	EntityMetadata map[string]interface{}
+	// EncodingIdentifier ...
+	EncodingIdentifier string
+	// InstanceIdentifier ...
+	InstanceIdentifier string
```

Comment:
Is there no documentation for this stuff? I believe this volume stuff is in behaviour packs so we could probably RE it

Reply:
I'm not exactly sure but I'll do some digging and update you if I find anything.

Reply:
This PR should be good to go once this is resolved.

Reply:
Resolved.